src/rollup/ab-test/init-ab-test.js

```diff
@@ -0,0 +1,53 @@
+/** @module pathfora/ab-test/init-ab-test */
+
+import saveCookie from '../utils/cookies/save-cookie';
+import readCookie from '../utils/cookies/read-cookie';
+
+import { pathforaDataObject } from '../globals/config';
```

Comment:
Import order is not an issue in rollup, because yeah it gets imported in the child module as well. I can try to create some sort of consistent order for imports though. Currently things are grouped in the same way that the folders are, which is why there are the random spaces you are seeing. Might make more sense if I just add comments for those groups though.